hw/drivers/nimble/nrf52/src/ble_phy.c

```diff
@@ -1824,6 +1833,28 @@ ble_phy_resolv_list_disable(void)
 }
 #endif
 
+#if MYNEWT_VAL(BLE_LL_DIRECT_TEST_MODE) == 1
+void ble_phy_enable_dtm(void)
+{
+    /* Disable whitening */
```

Comment:
some comment why you need this would be nice

Reply:
yup, as per spec. Added.